packages/teams-js/src/public/media.ts

```diff
@@ -264,31 +289,34 @@ export namespace media {
         assembleAttachment: [],
       };
       const params = [actionName, this.content];
-      this.content && callback && sendMessageToParent('getMedia', params);
+      this.content && !isNullOrUndefined(callback) && sendMessageToParent('getMedia', params);
```

Comment:
>isNullOrUndefined [](http://example.com/codeflow?start=23&length=17)

Without this, the violation is triggered because it thinks this code is trying to invoke/call the `callback` function